internal/qerr/error_codes.go

```diff
@@ -52,9 +51,10 @@ func (e ErrorCode) String() string {
 		return "PROTOCOL_VIOLATION"
 	case InvalidMigration:
 		return "INVALID_MIGRATION"
-	case CryptoError:
-		return "CRYPTO_ERROR"
 	default:
+		if e >= 0x100 && e < 0x200 {
+			return fmt.Sprintf("CRYPTO_ERROR %d", e-0x100)
```

Comment:
Does it make sense to print these in decimal if the spec is hexa?

Reply:
TLS error codes are decimal, but it makes sense for the "unknown error code" message.